src/client/pydaos/pydaos_core.py

```diff
@@ -35,7 +35,7 @@
 
 from . import DAOS_MAGIC
 from . import PyDError
-from . import dc
+from . import DaosClient
```

Comment:
(pylint-import-error) Unable to import 'pydaos' (inconsistent use of tabs and spaces in indentation (<string>, line 73))